### PR TITLE
fix: typos

### PR DESCRIPTION
**Pull request by omahs**
Fix: typos

### CODE REVIEW

File: packages/docs/docs/how-to-use.mdx [hunk 1]
```diff
@@ -17,7 +17,7 @@ You can try this code directly in the developer console.
 ## Detect when fuel is loaded
 
 It's possible that your application loads before `window.fuel` is injected. To detect when the `fuel` is loaded and ready to use, you can listen to the event `FuelLoaded` on the document.
-If you are using react take a look on the source code of the example bellow to see how we implement it.
+If you are using react take a look at the source code of the example below to see how we implement it.
```
Comment (tomiiide) [inline packages/docs/docs/how-to-use.mdx:20]:
```suggestion
If you are using react, take a look at the source code of the example below to see how we implement it.
```

**Review by tomiiide — CHANGES_REQUESTED** [review]
Looks good, just a little typo improvement.

**Comment by omahs** [comment]
Just updated!

**Review by LuizAsFight — APPROVED** [review]

**Comment by omahs** [comment]
@luizstacio It seems that the files were updated already and typos were fixed so I'll close this :)

**Pull request closed without merging**